tests/psmdbcluster/psmdb_cluster_test.go

```diff
@@ -85,6 +88,36 @@ func TestPSMDBClusterAPI(t *testing.T) {
 	}
 	assert.True(t, clusterFound)
 
+	t.Log("Wating for cluster to be ready")
+	err = waitForPSMDBClusterState(tests.Context, kubeconfig, name, controllerv1beta1.PSMDBClusterState_PSMDB_CLUSTER_STATE_READY)
+	require.NoError(t, err)
+
+	updateReq := &controllerv1beta1.UpdatePSMDBClusterRequest{
+		KubeAuth: &controllerv1beta1.KubeAuth{
+			Kubeconfig: kubeconfig,
+		},
+		Name: name,
+		Params: &controllerv1beta1.PSMDBClusterParams{
+			ClusterSize: 6,
+			Replicaset: &controllerv1beta1.PSMDBClusterParams_ReplicaSet{
+				ComputeResources: &controllerv1beta1.ComputeResources{},
+				DiskSize:         1024 * 1024 * 1024,
```

Comment:
We can't update disk size.

Reply:
The field has been made mandatory in another PR
```
psmdb_cluster_test.go:114:
                Error Trace:    psmdb_cluster_test.go:114
                Error:          Received unexpected error:
                                rpc error: code = InvalidArgument desc = invalid field Params.Replicaset.DiskSize: value '0' must be greater than '0'
                Test:           TestPSMDBClusterAPI
```

Reply:
Yes, but you extracted Update message and it should not be. If it is, then it should be fixed.